liquibase-standard/src/main/java/liquibase/command/core/helpers/LockServiceCommandStep.java

```diff
@@ -45,7 +45,9 @@ public String[][] defineCommandNames() {
     @Override
     public void cleanUp(CommandResultsBuilder resultsBuilder) {
         try {
-            LockServiceFactory.getInstance().getLockService(database).releaseLock();
```

Comment:
As you removed the usage of the class field database, could you also move declaration to `run ` method and remove it from class level? 
Thanks